luisy/tasks/base.py

```diff
@@ -247,3 +247,9 @@ class DatabricksTask(Task):
     @property
     def spark(self):
         return Config().spark
+
+    def write(self, obj):
+        target = self.output()
+        if isinstance(target, LocalTarget):
+            obj = obj.toPandas()
```

Comment:
Do we have to make sure that `obj` is a spark dataframe? Could be that the user (accidentially) already did this in his `run` method.